tcms/testruns/static/testruns/js/get.js

```diff
@@ -241,6 +256,24 @@ function renderTestExecutionRow(template, testExecution, testExecutionStatus) {
     return template
 }
 
+function changeStatusBulk(statusId) {
+    const selected = selectedCheckboxes()
+    if ($.isEmptyObject(selected)) {
+        return false
+    }
+
+    selected.executionIds.forEach(executionId => {
+        jsonRPC('TestExecution.update', [executionId, {
+            'status': statusId,
+            // TODO
+            // 'tested_by': Nitrate.User.pk
```

Comment:
@atodorov do we still want to keep this behaviour? if yes, then we would need something like: https://github.com/kiwitcms/Kiwi/blob/master/tcms/templates/tcms_base.html#L40-L50

Reply:
The behavior is "whomever changes the status is also assigned as default_tester" and it should be kept. However I think this is better to be done on the API side. TestExecution.update already contains other special if conditions so checking if status is set and updating default_tester seems logical. It will also allow for easier testing of this behavior and will benefit external clients & plugins.